tardis/plasma/standard_plasmas.py

```diff
@@ -175,15 +180,33 @@ def assemble_plasma(config, model, atom_data=None):
         property_kwargs[MarkovChainTransProbsCollector] = {
             "inputs": transition_probabilities_outputs
         }
-        if config.plasma.nlte_ionization_species:
-            nlte_ionization_species = config.plasma.nlte_ionization_species
-            for species in nlte_ionization_species:
-                if not (species in config.plasma.continuum_interaction.species):
-                    raise PlasmaConfigError(
-                        f"NLTE ionization species {species} not in continuum species."
-                    )
+        if (
+            config.plasma.nlte_ionization_species
+            or config.plasma.nlte_excitation_species
+        ):
```

Comment:
Why is this if clause needed? You later do individual checks `if config.plasma.nlte_ionization_species` and `if config.plasma.nlte_excitation_species` anyways.

Reply:
As I am saving them in the same plasma property, I added this part to make sure that that property is added no matter if one of them is empty or not.

Reply:
Thanks for clarifying. That makes sense. I guess you could still get rid of the if clause if you do the the assignments `nlte_ionization_species = config.plasma.nlte_ionization_species` and `nlte_excitation_species = config.plasma.nlte_excitation_species` regardless of whether they are actually needed. However, I don't think this a much better solution.

Reply:
so, I got rid of those 2 lines, and the code still seems to be working.

Reply:
I got rid of the reassignment for `nlte_ionization_species` and `nlte_excitation_species`. Is there anything else I should change?

Reply:
I think you need to move the assignments `nlte_ionization_species = config.plasma.nlte_ionization_species` and `nlte_excitation_species = config.plasma.nlte_excitation_species` out of the inner two if clauses. Otherwise, these variables are not defined if they the values from the config are empty lists. Alternatively, you can also use `config.plasma.nlte_excitation_species` and `config.plasma.nlte_ionization_species` in line 206/207.